.github/workflows/test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Kindly make sure that your .gitignore file is ignoring the test.md file. Maybe the problem is the path inside of your .gitignore file.  Another thing you could try is moving the test.md file to the root path of your project. :nerd_face: :nerd_face: 
![image](https://user-images.githubusercontent.com/98363075/197655107-958e70f3-0924-469e-87b8-83d01ee50be2.png)